static/app/views/performance/transactionDetails/styles.tsx

```diff
@@ -21,7 +21,7 @@ export function MetaData({
   badge,
 }: MetaDataProps) {
   return (
-    <HeaderInfo>
+    <HeaderInfo data-test-id="meta-data">
```

Comment:
instead of a `data-test-id`, shall we look for the rendered text... like

`screen.getByRole('heading', {name: 'Trace Navigator'})`?

Reply:
Yes good call, let's do that instead